### ISSUE TITLE
aeld.js don't work on firefox sometimes

### ISSUE THREAD

**Issue by yingziwu** [uBlockOrigin/uBlock-issues#1821]
### Prerequisites

- [X] I verified that this is not a filter issue (MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues))
- [X] This is not a [support issue or a question](https://old.reddit.com/r/uBlockOrigin/)
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue
- [X] The issue is not present after wholly disabling uBlock Origin ("uBO") in the browser
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### I tried to reproduce the issue when...

- [X] uBO is the only extension
- [X] uBO with default lists/settings
- [X] using a new, unmodified browser profile

### Description

The aeld.js don't work on firefox sometimes.

### A specific URL where the issue occurs

https://blog.bgme.me/ublock-test/test.html

### Steps to Reproduce

1. Open firefox.
2. Add the following rule to `My filters`.
```
blog.bgme.me##+js(aeld, click, /popMagic|adConfig/)
blog.bgme.me##+js(aeld, mouseover, mouseover)
blog.bgme.me##+js(aell)
```
3. Open https://blog.bgme.me/ublock-test/test.html
4. Open Console and click `More information...` link.

### Expected behavior

The event `click` and `mouseover` are blocked.

### Actual behavior

On Chromium,  the both events are blocked.

But on firefox, the event of `mouseover` is blocked, but `click` is not blocked.

### uBlock Origin version

1.38.6

### Browser name and version

Firefox 94.0.1 and Chromium 96.0.4664.45

### Operating System and version

Arch Linux, Linux 5.15.2-zen1-1-zen

**Comment by uBlock-user** [comment]
Duplicate of https://github.com/uBlockOrigin/uBlock-issues/issues/1820

**Comment by gwarser** [comment]
Both are blocked on my side on dirty profile in Nightly, but does not in clean Firefox 94 or clean Nightly. Looks like racing, but I will rather this it will be other way around - not applied on dirty profile.
